packages/js/create-woo-extension/index.js

```diff
@@ -1,21 +1,23 @@
-module.exports = {
-	templatesPath: __dirname,
-	defaultValues: {
-		npmDependencies: [
-			'@wordpress/hooks',
-			'@wordpress/i18n',
-			'@woocommerce/components',
-		],
-		npmDevDependencies: [
-			'@woocommerce/dependency-extraction-webpack-plugin',
-			'@woocommerce/eslint-plugin',
-			'@wordpress/prettier-config',
-			'@wordpress/scripts@24.6.0',
-		],
-		namespace: 'extension',
-		license: 'GPL-3.0+',
-		customScripts: {
-			postinstall: 'composer install',
-		},
-	},
-};
+#!/usr/bin/env node
+const path = require('node:path');
+const { Command } = require('commander');
+const slug = require('slug');
+const spawn = require('child_process').spawn;
+const package_info = require('./lib/package-info');
```

Comment:
JS tends to favor camelCase, ie `packageInfo`. We should stick to this convention here also

Reply:
Right, thanks for the catch. :ok_hand: